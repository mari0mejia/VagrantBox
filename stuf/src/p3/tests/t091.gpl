// when a double is passed to random, it should be rounded down
//
int a = random( 7 );
int b = random( 12.5 );
double d = random( 7 );
double e = random( 12.5 );
string g = random( 7 );
string h = random( 12.5 );
