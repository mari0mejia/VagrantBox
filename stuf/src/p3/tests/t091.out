gpl.cpp::main()
  input file(tests/t091.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
int a = 5
int b = 0
double d = 3
double e = 9
string g = "0"
string h = "10"
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
